core/chains/evm/config/chain_scoped.go

```diff
@@ -126,9 +126,10 @@ func (e *EVMConfig) BlockEmissionIdleWarningThreshold() time.Duration {
 
 func (e *EVMConfig) ChainType() commonconfig.ChainType {
 	if e.C.ChainType == nil {
-		return ""
+		return commonconfig.ChainTypeNone
 	}
-	return commonconfig.ChainType(*e.C.ChainType)
+	chainType, _ := commonconfig.ChainTypeFromSlug(*e.C.ChainType)
```

Comment:
Ignored error

Reply:
Is the linter broken again?

Reply:
Does this error need to be handled here? The signature of `ChainType()` doesn't return an error and there isn't really a way to properly handle it here.

I think we could operate under the assumption that this error had already been checked during startup, as it is used and checked during the config validatation. Even if the node starts, it's going to return `ChainTypeNone` here, which will just do nothing.

Reply:
Embedding the type in the config struct should eliminate this issue.

Reply:
Embedded the type directly into the config struct here: https://github.com/smartcontractkit/chainlink/pull/13272/files#diff-7cc03cd5adaef6b2b6e25f221cf16163cbd44f0b8f2bf188febaf01389a7bf9eR344
The workaround with the second type is needed to preserve the original value, making it available for deprecation checks